### PR TITLE
feat(ibmmq): windows release

### PR DESCRIPTION
**Pull request by paologallinaharbur**
Tested locally and triggered the package creation with:
 - https://github.com/newrelic/newrelic-prometheus-exporters-packages/actions/runs/3847523880/jobs/6554158341
 -

### CODE REVIEW

File: .github/workflows/manual-artifacts-generation.yml [hunk 1]
```diff
@@ -25,6 +25,10 @@ jobs:
         with:
           version: v1.13.1
           install-only: true
+      - name: Install dependencies
+        run: |
+          sudo apt-get install rpm ruby ruby-dev rubygems build-essential rpm gnupg2 gpg-agent debsigs
+          sudo gem install --no-document fpm
```
Comment (sigilioso) [inline .github/workflows/manual-artifacts-generation.yml:31]: Thank you! 🙌

File: exporters/ibmmq/exporter.yml [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
+version: 0.4.0
 # Relative path to the License path from the repository root
 exporter_license_path: LICENSE
 # URL to the git project hosting the exporter
 exporter_repo_url: https://github.com/ibm-messaging/mq-metric-samples
 # Tag of the exporter to checkout
 exporter_tag:
 # Commit of the exporter to checkout (used if tag property is empty)
-exporter_commit: d268624ff90d2fbbec18d62b6f73ed8955c4ee8f
+exporter_commit: 322e8f6cc5ac687e5766acd4eaee447cc7d62d7a
```
Comment (sigilioso) [inline exporters/ibmmq/exporter.yml:12]: You took the chance to update the exporter, right? Nice!

**Review by sigilioso — APPROVED** [review]
👍

File: exporters/ibmmq/exporter.yml (hunk 2 above)
Comment (paologallinaharbur) [reply exporters/ibmmq/exporter.yml:12]: yes, we were pointing to an off-tag release and now we are back on track

**Pull request merged**